backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -16,7 +16,7 @@ To back up Kubernetes resources and internal images, you must have object storag
 * xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc#installing-oadp-azure[Microsoft Azure]
 * xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc#installing-oadp-gcp[Google Cloud Platform]
 * xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc#installing-oadp-mcg[Multicloud Object Gateway]
-* S3-compatible object storage, such as Noobaa or Minio
+* AWS S3 compatible object storage, such as Noobaa or Minio
```

Comment:
@ShaunaDiaz 

Thank you for your feedback.

You are correct that Amazon calls this product as you did.

However, the RedHat style guide, the rest of this document and many others refer to it as AWS S3. 

https://redhat-documentation.github.io/supplementary-style-guide/#project

For the sake of consistency and to prevent confusing the reader, I would like to stay with the phrase AWS S3 that is used in our style guide.

Thanks,
Carmi

Reply:
@CarmiWisemon Peer review suggestions are just that--suggestions. You're free to follow your instincts on anything that's not spelled out in the guidance somewhere. No worries. 👍 

I doubt the suggestion would be confusing, but "S3" is used heavily as you have here throughout the docs.